scripts/dispatch.sh

```diff
@@ -2,5 +2,5 @@
 
 curl -H "Authorization: token $GH_TOKEN" \
 --request POST \
---data '{"event_type": "publish-engines", "client_payload": {"branch": "master", "commit": "0c2898954d761d1c92f304ff1b7917c601c2e3d8"}}' \
+--data '{"event_type": "publish-engines", "client_payload": {"branch": "main", "commit": "0c2898954d761d1c92f304ff1b7917c601c2e3d8"}}' \
 https://api.github.com/repos/prisma/engines-wrapper/dispatches
```

Comment:
What does this do? Internal trigger of a run?

Reply:
It's an example to do a "dispatch" locally using your bash terminal